lib/astarte_data_access/config.ex

```diff
@@ -38,4 +57,162 @@ defmodule Astarte.DataAccess.Config do
     os_env: "CASSANDRA_NODES",
     type: CQExNodes,
     default: [{"localhost", 9042}]
+
+  @envdoc """
+  The Xandra authenticator that implements support for PasswordAuthenticator to authenticate
+  with Cassandra server.
+  """
+  app_env :xandra_authentication_module, :astarte_data_access, :xandra_authentication_module,
+    os_env: "XANDRA_AUTHENTICATION_MODULE",
+    type: :module,
+    default: Xandra.Authenticator.Password
+
+  @envdoc """
+  The CQEx authenticator that implements support for PasswordAuthenticator to authenticate
+  with Cassandra server.
+  """
+  app_env :cqex_authentication_module, :astarte_data_access, :cqex_authentication_module,
+    os_env: "CQEX_AUTHENTICATION_MODULE",
+    type: :module,
+    default: :cqerl_auth_plain_handler
+
+  @envdoc "The username used to log into cassandra. Defaults to 'cassandra'."
+  app_env :cassandra_username, :astarte_data_access, :cassandra_username,
+    os_env: "CASSANDRA_USERNAME",
+    type: :binary,
+    default: "cassandra"
+
+  @envdoc "The password used to log into cassandra. Defaults to 'cassandra'."
+  app_env :cassandra_password, :astarte_data_access, :cassandra_password,
+    os_env: "CASSANDRA_PASSWORD",
+    type: :binary,
+    default: "cassandra"
+
+  @envdoc """
+  Whether or not results of and parameters to Xandea.execute/4 will have atom keys.
+  Defaults to false.
+  """
+  app_env :atom_keys, :astarte_data_access, :atom_keys,
+    os_env: "CASSANDRA_ATOM_KEYS_ENABLED",
+    type: :boolean,
+    default: "false"
```

Comment:
Same as above (also, we don't want someone to put this to `true`, otherwise all hell breaks lose because we match on string keys _and_ we start leaking atoms)